admin/system/upload-images.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/***************************************************
+ * Only these origins are allowed to upload images *
+ ***************************************************/
+$accepted_origins = array("http://localhost:8000", "http://192.168.1.1", "http://localhost/pindad-clone", "localhost/pindad-clone");
+
+/*********************************************
+ * Change this line to set the upload folder *
+ *********************************************/
+$imageFolder = "../../news/images/";
+
+reset ($_FILES);
+$temp = current($_FILES);
+if (is_uploaded_file($temp['tmp_name'])){
+  if (isset($_SERVER['HTTP_ORIGIN'])) {
+    // same-origin requests won't set an origin. If the origin is set, it must be valid.
+    if (in_array($_SERVER['HTTP_ORIGIN'], $accepted_origins)) {
+      header('Access-Control-Allow-Origin: ' . $_SERVER['HTTP_ORIGIN']);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)